qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/model.py

```diff
@@ -565,15 +565,7 @@ def validate_x_limits(self, users_limits):
         else:
             self.x_min = self.data_x_min
             self.x_max = self.data_x_max
-            success = self.determine_x_limits()
```

Comment:
Is this `self.determine_x_limits` used anywhere else? Is it possible to delete it?